lib/devise-guests/controllers/helpers.rb

```diff
@@ -80,6 +81,10 @@ def guest_#{mapping}_unique_suffix
           Devise.friendly_token + "_" + Time.now.to_i.to_s + "_" + unique_#{mapping}_counter.to_s
         end
 
+        def set_guest_#{mapping}_params
```

Comment:
I haven't checked if devise etc have any conventions around this, but because this method is just providing the parameters, I think dropping the `set_` prefix is appropriate? 

```suggestion
        def guest_#{mapping}_params
```